.buildkite/pipeline.yml

```diff
@@ -194,7 +194,172 @@ steps:
 
   - label: "quark-test on fedora 42(beta)"
     key: test_fedora_42
-    command: "./.buildkite/runtest_fedora.sh 42"
+    command: "./.buildkite/runtest_distro.sh fedora 42"
+    depends_on:
+      - make_docker
+    agents:
+      image: family/core-ubuntu-2204
+      provider: gcp
+      machineType: n2-standard-2
+      enableNestedVirtualization: true
+
+  - label: "quark-test on rhel 8"
+    key: test_rhel_8
+    command: "./.buildkite/runtest_distro.sh rhel 8"
+    depends_on:
+      - make_docker
+    agents:
+      image: family/core-ubuntu-2204
+      provider: gcp
+      machineType: n2-standard-2
+      enableNestedVirtualization: true
+
+  - label: "quark-test on rhel 8.3"
+    key: test_rhel_8_3
+    command: "./.buildkite/runtest_distro.sh rhel 8.3"
+    depends_on:
+      - make_docker
+    agents:
+      image: family/core-ubuntu-2204
+      provider: gcp
+      machineType: n2-standard-2
+      enableNestedVirtualization: true
+
+  - label: "quark-test on rhel 8.4"
+    key: test_rhel_8_4
+    command: "./.buildkite/runtest_distro.sh rhel 8.4"
+    depends_on:
+      - make_docker
+    agents:
+      image: family/core-ubuntu-2204
+      provider: gcp
+      machineType: n2-standard-2
+      enableNestedVirtualization: true
+
+  - label: "quark-test on rhel 8.5"
+    key: test_rhel_8_5
+    command: "./.buildkite/runtest_distro.sh rhel 8.5"
+    depends_on:
+      - make_docker
+    agents:
+      image: family/core-ubuntu-2204
+      provider: gcp
+      machineType: n2-standard-2
+      enableNestedVirtualization: true
+
+  - label: "quark-test on rhel 8.6"
+    key: test_rhel_8_6
+    command: "./.buildkite/runtest_distro.sh rhel 8.6"
+    depends_on:
+      - make_docker
+    agents:
+      image: family/core-ubuntu-2204
+      provider: gcp
+      machineType: n2-standard-2
+      enableNestedVirtualization: true
+
+  - label: "quark-test on rhel 8.7"
+    key: test_rhel_8_7
+    command: "./.buildkite/runtest_distro.sh rhel 8.7"
+    depends_on:
+      - make_docker
+    agents:
+      image: family/core-ubuntu-2204
+      provider: gcp
+      machineType: n2-standard-2
+      enableNestedVirtualization: true
+
+  - label: "quark-test on rhel 8.8"
+    key: test_rhel_8_8
+    command: "./.buildkite/runtest_distro.sh rhel 8.8"
+    depends_on:
+      - make_docker
+    agents:
+      image: family/core-ubuntu-2204
+      provider: gcp
+      machineType: n2-standard-2
+      enableNestedVirtualization: true
+
+  - label: "quark-test on rhel 8.9"
+    key: test_rhel_8_9
+    command: "./.buildkite/runtest_distro.sh rhel 8.9"
+    depends_on:
+      - make_docker
+    agents:
+      image: family/core-ubuntu-2204
+      provider: gcp
+      machineType: n2-standard-2
+      enableNestedVirtualization: true
+
+  - label: "quark-test on rhel 9"
```

Comment:
what the difference between this and rhel 9.0? perhaps the script will show me.

Reply:
9 is the current 9 mainline, 9.0 is the original 9.0 release.
5.14.0-503.35.1.el9_5.x86_64
vs
4.18.0-425.3.1.el8.x86_64

Reply:
ooops, just realized the URL for 8.x and 9.x was incorrect :(
812325c